proto/cosmos/staking/v1beta1/query.proto

```diff
@@ -184,7 +184,7 @@ message QueryValidatorDelegationsResponse {
 // Query/ValidatorUnbondingDelegations RPC method
 message QueryValidatorUnbondingDelegationsRequest {
   // validator_addr defines the validator address to query for.
-  string validator_addr = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string validator_addr = 1 [(cosmos_proto.scalar) = "cosmos.ValidatorAddressString"];
```

Comment:
What's the effect of this, in other words, where do we consume this changed option? Looking at generated protos only the packed message descriptors have changed.

Reply:
Here: https://github.com/cosmos/cosmos-sdk/blob/main/client/v2/autocli/flag/field.go#L159 and in the builder.go file changes in this PR